HOST lewis.lbl.gov
only_i 222
only_code None
force_amber_prep True
dry_run False
... 222 3nxu
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/nx/3nxu
3nxu.pdb 4phenix_3nxu.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/nx/pdb3nxu.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/nx/3nxu/3nxu.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/nx/pdb3nxu.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/nx/3nxu/3nxu.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 3nxu.pdb
===================================================

----------Chains
The following (original) chains have been found:
A
B

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
None

---------- Gaps (Renumbered Residues!)
gap of 9.542185 A between THR 236 and ARG 237
gap of 6.978612 A between ASN 249 and ALA 250
gap of 9.844116 A between THR 693 and ARG 694
gap of 8.177084 A between ASN 706 and ALA 707

---------- Mising heavy atom(s)

None
not prmtop_filename /net/lewis/scratch1/nigel/amber/hi_lo/nx/3nxu/4amber_3nxu.prmtop
not prmtop_filename /net/chevy/raid1/nigel/amber/hi_lo/nx/3nxu/4amber_3nxu.prmtop

	Running AmberPrep : 3nxu.pdb


==================================================
Running pdb4amber on 3nxu.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================

==================================================
Running elbow/antechamber for HEM 
==================================================
	Difficulties in the ring structures may lead to problems later

   Residue has metal at the centre of a coordination sphere. Using input
   geometry to generate restraints. Hydrogens may not be added.
   

MoleculeClass :  C:34  N: 4  O: 4 Fe: 1 (CHEMICAL COMPONENTS format)
	73 atoms
	80 bonds
	144 angles
	218 dihedrals
	1 rings (groups, rings, atoms) 8 44
	0 chirals

~> antechamber -i 4antechamber_HEM.pdb -fi pdb -o HEM.mol2 -fo mol2       -nc -2 -m 1 -s 2 -pf y -c bcc -at gaff2 -ek "qm_theory='AM1',grms_tol=0.0005,scfconv=1.d-10,maxcyc=0,ndiis_attempts=700,"

(2) adjust atom valence penalty parameters in APS.DAT, and/or 
Sorry: Error: cannot run "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/amber/bin/sqm -O -i sqm.in -o sqm.out" of bcc() in charge.c properly, exit
